lib/zendesk_apps_tools/command.rb

```diff
@@ -73,11 +75,18 @@ def validate
         end
         say_error_and_exit error
       end
+
       valid = errors.none?
 
       if valid
-        app_package.warnings.each { |w| say w.to_s, :yellow }
-        say_status 'validate', 'OK'
+        require 'zendesk_apps_support/app_version'
+        @zas ||= ZendeskAppsSupport::AppVersion.new(manifest.version)
+
+        if @zas.valid_for_update?
+          app_package.warnings.each { |w| say w.to_s, :yellow }
+          say_status 'validate', 'OK'
+        end
+        sunsetted app_package.manifest.framework_version
```

Comment:
`manifest.framework_version` was. But at the moment it's failing tests and I'm trying to work out why.